tests/nightly/jenkins-nightly-run.sh

```diff
@@ -45,7 +46,8 @@ testcases=("${@:-${DEFAULT_TESTCASES[@]}}")
 # TODO: the version downloaded by this logic is not coupled with the tests that will be run against it. This should be altered to pull a version that matches the commit SHA of the tests
 # we will be running or similar mechanism.
 VCH_BUILD=${VCH_BUILD:-${DEFAULT_VCH_BUILD}}
-input=$(gsutil ls -l gs://vic-engine-builds/${VIC_BINARY_PREFIX}${VCH_BUILD} | grep -v TOTAL | sort -k2 -r | head -n1 | xargs | cut -d ' ' -f 3 | cut -d '/' -f 4)
+VCH_BRANCH=${VCH_BRANCH:-${DEFAULT_VCH_BRANCH}}/
+input=$(gsutil ls -l gs://vic-engine-builds/${VCH_BRANCH#/}${VIC_BINARY_PREFIX}${VCH_BUILD} | grep -v TOTAL | sort -k2 -r | head -n1 | xargs | cut -d ' ' -f 3 | cut -d '/' -f 4)
```

Comment:
A `/` between branch and filename rather than in the variable value? I found this construction confusing.

Reply:
This weird pattern is to allow `VCH_BRANCH` to be empty (for when we're pulling a build from the top-level directory). Perhaps this script doesn't need to be backwards compatible, but it seemed safer to make it so.

Reply:
Is this clearer?
```
VCH_BRANCH=${VCH_BRANCH:-${DEFAULT_VCH_BRANCH}}
input=$(gsutil ls -l gs://vic-engine-builds/${VCH_BRANCH:+${VCH_BRANCH}/}${VIC_BINARY_PREFIX}${VCH_BUILD} | grep -v TOTAL | sort -k2 -r | head -n1 | xargs | cut -d ' ' -f 3 | cut -d '/' -f 4)
```